### DISCUSSION TITLE
Import data into table

### DISCUSSION THREAD

**Discussion by 467481** [planetscale/discussion#372] [Bug Reports] [answered]
Hi, 

For my project I am trying to import data from a csv file into my table using the MYSQL workbench import wizard. However, I get the error "Error Code: 1105 BUG: unexpected statement type: *sqlparser.PrepareStmt". I think this has something to do with planetscale because I also tried to import the same csv file into a docker mysql container and that did work. Does anyone know how I can fix this issue?

**Comment by orware** [comment, marked as answer]
Hello @467481,

I was able to confirm the same error/behavior within MySQL Workbench when attempting to use the Table Data Import Wizard feature.

At the moment I'm unsure as to what statement is being attempted that runs into the error so I will collect some additional details on my end and submit a bug request so that can be investigated further and potentially fixed on our end, since it may be a bug within our Vitess layer.

However, I can offer the suggestion of using DBeaver's Import Data feature as an alternative, which allows you to import a CSV file too, and based on my test a few moments ago, it didn't seem to run into the same issue that MySQL Workbench was encountering:
![image](https://user-images.githubusercontent.com/931650/211930221-0dbadba7-ff9a-42a5-af04-664e405f1f27.png)
